tests/unit/python/foglamp/common/test_statistics.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+from unittest.mock import MagicMock, patch
+import pytest
+
+from foglamp.common.statistics import Statistics, _logger
+from foglamp.common.storage_client.storage_client import StorageClient
+
+
+__author__ = "Ashish Jabble"
+__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+
+@pytest.allure.feature("unit")
+@pytest.allure.story("common", "statistics")
+class TestStatistics:
+
+    def test_init_with_storage(self):
+        storage_client_mock = MagicMock(spec=StorageClient)
+        s = Statistics(storage_client_mock)
+        assert isinstance(s, Statistics)
+        assert isinstance(s._storage, StorageClient)
+
+    def test_init_with_no_storage(self):
+        storage_client_mock = None
+        with pytest.raises(TypeError) as excinfo:
+            Statistics(storage_client_mock)
+        assert 'Must be a valid Storage object' == str(excinfo.value)
+
+    async def test_update(self):
+        storage_client_mock = MagicMock(spec=StorageClient)
+        s = Statistics(storage_client_mock)
+        payload = '{"where": {"column": "key", "condition": "=", "value": "READING"}, ' \
+                  '"expressions": [{"column": "value", "operator": "+", "value": 5}]}'
+        result = {"response": "updated", "rows_affected": 1}
```

Comment:
better to rename it as expected or expected_result. in general its for actual result